common/icons/package.json

```diff
@@ -2,11 +2,13 @@
   "name": "@weco/icons",
   "description": "Icon set and optimisation for wellcomecollection.org",
   "license": "MIT",
+  "sideEffects": false,
```

Comment:
I've not seen this before – what's it do?

Reply:
https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

it a weird one, I don't think it should be necessary here but when I added it it knocked a couple of kb off the bundle 🤷